icons/orbit.svg

```diff
@@ -9,9 +9,9 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <circle cx="12" cy="12" r="3" />
-  <circle cx="18" cy="6" r="2" />
-  <circle cx="6" cy="18" r="2" />
-  <path d="M18.8,7.8c0.2,0.3,0.4,0.7,0.5,1.1c0.4,1,0.6,2,0.6,3.1s-0.2,2.2-0.6,3.1c-0.4,1-1,1.8-1.7,2.5 s-1.6,1.3-2.5,1.7c-1,0.4-2,0.6-3.1,0.6" />
-  <path d="M5.2,16.2c-0.2-0.3-0.4-0.7-0.5-1.1c-0.4-1-0.6-2-0.6-3.1s0.2-2.2,0.6-3.1c0.4-1,1-1.8,1.7-2.5S7.9,5,8.9,4.6 c1-0.4,2-0.6,3.1-0.6" />
+  <circle cx="12" cy="12" r="3"/>
+  <circle cx="18" cy="6" r="2"/>
+  <circle cx="6" cy="18" r="2"/>
+  <path d="M18.8 7.8A8 8 0 0 1 12 20"/>
+  <path d="M5.2 16.2A8 8 0 0 1 12 4"/>
```

Comment:
Ouch, those were some funky lines. 😩 

The arcs are (still) a bit longer than they should be though:
```suggestion
  <circle cx="12" cy="12" r="3" />
  <circle cx="18" cy="6" r="2" />
  <circle cx="6" cy="18" r="2" />
  <path d="M12 20a8 8 0 0 0 6.82-12.17" />
  <path d="M12 4a8 8 0 0 0-6.82 12.17" />
```